.github/workflows/uninstall_install_sandbox.yaml

```diff
@@ -28,15 +28,28 @@ jobs:
           namespace: ${{ secrets.OPENSHIFT_NAMESPACE }}
       
       - name: Uninstall Traction via Helm
-        continue-on-error: true
+        continue-on-error: false
         run: |
           helm uninstall traction-sandbox -n ${{ secrets.OPENSHIFT_NAMESPACE }} --wait --timeout=20m
 
       - name: Remove Traction Openshift Objects
-        continue-on-error: true 
+        continue-on-error: false 
         run: |
           oc delete -n ${{ secrets.OPENSHIFT_NAMESPACE }} all,secret,pod,networkpolicy,configmap,pvc --selector "app.kubernetes.io/instance"=traction-sandbox
 
+      - name: Rocket.Chat Notification
+        uses: RocketChat/Rocket.Chat.GitHub.Action.Notification@master
+        if: always()
+        with:
+          type:  ${{ job.status }}
+          job_name: '*Uninstall Traction Sandbox*'
+          mention: 'here'
+          mention_if: 'failure'
+          channel: '#ditp-gha-notifications'
+          url: ${{ secrets.ROCKETCHAT_WEBHOOK }}
+          commit: true
+          token: ${{ secrets.ROCKETCHAT_TOKEN }}
```

Comment:
`token: ${{ secrets.ROCKETCHAT_TOKEN }}` is not required if the `secrets.ROCKETCHAT_WEBHOOK` contains the token, which in our case it does.  Refer to your previous work with this action here; https://github.com/bcgov/von-bc-registries-audit/blob/main/.github/workflows/actions/deploy/action.yml#L83-L93